scripts/assets/zip/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>CKEditor 5 - Quick start ZIP</title>
+		<link rel="stylesheet" href="./ckeditor5/ckeditor5.css">
+        <style>
+            .main-container {
+                width: 795px;
+                margin-left: auto;
+                margin-right: auto;
+            }
+        </style>
+    </head>
+    <body>
+        <div class="main-container">
+            <div id="editor">
+                <p>Hello from CKEditor 5!</p>
```

Comment:
This sample is very poor. I would copy the Builder's output here to help users discover the product and where to seek support, including our issue tracker.

Reply:
The minimalistic sample was done on purpose. So that someone is not too attached to it, but yes, redirecting to builder needs to be done here, I'm still thinking about it.